runtime/vm/mz64/UpcallThunkGen.cpp

```diff
@@ -287,6 +287,7 @@ createUpcallThunk(J9UpcallMetaData *metaData)
 	}
 	for (I_32 i = 0; i < lastSigIdx; i++) {
 		U_32 argSizeInBytes = sigArray[i].sizeInByte;
+		const int totalStackSlotsReq = ((argSizeInBytes - 1) / 8) + 1;
```

Comment:
Is it possible that `argSizeInBytes` is zero? If so, a better expression is
```c
		const I_32 totalStackSlotsReq = (argSizeInBytes + 7) / 8;
```
Either way, the type should be `I_32` instead of `int`.

Reply:
I'm not sure if there's any realistic scenario where argument size would be 0 or if it would make sense for it to ever be 0. Though I suppose it's technically possible for that field to be assigned a value of 0. In any case, I think we will get the same result with both solutions so I'll go ahead and use your suggestion.

Reply:
Addressed in https://github.com/eclipse-openj9/openj9/commit/cd423b0e112a5d4671243b7cc6f50f26d0ecb8a2.